CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Workflow : Variables now have a pretty type to display on frontend
 - WorkflowState: add memory usage monitoring & markdown
 - Workflow : add documentation to workflow when export in .py
+- (forms) : Update with last volmdlr
```

Comment:
There is not need to set topic between parenthesis, plus, this should be kept in alphabetical order